### PR TITLE
Scratch cheatsheet

### PR DESCRIPTION
**Pull request by preemeijer**
What does your Instant Answer do?
Scratch cheatsheet for the buildingblocks

What is the data source for your Instant Answer? (Provide a link if possible)
http://www.dummies.com/how-to/content/scratch-for-kids-for-dummies-cheat-sheet.html
http://wiki.scratch.mit.edu/wiki/Scratch_Wiki:Table_of_Contents/Program

Why did you choose this data source?
It was the best that I could find for this purpose

Are there any other alternative (better) data sources?
Yes used the original wiki for the value texts

What are some example queries that trigger this Instant Answer?
scratch cheatsheet

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)
For students who get some lessons with this program; starting programmers.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?
https://duck.co/ideas/idea/6332/scratch-cheatsheet

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
scratch-cheatsheet-screenshot

-----
IA Page: https://duck.co/ia/view/scratch_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/scratch.json [hunk 1]
```diff
@@ -0,0 +1,576 @@
+{
+   "id" : "scratch_cheat_sheet",
+   "name" : "Scratch Cheat Sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/scratch.json]: You can remove "Cheat Sheet" from the name

File: share/goodie/cheat_sheets/json/scratch.json [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+   "metadata" : {
+      "sourceName" : "Scratch wiki",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/scratch.json]: nit: I would capitalize Wiki

**Comment by moollaza** [comment]
@preemeijer this PR is looking much better! I've made a couple minor suggestions, but otherwise this is looking pretty good@

**Comment by daxtheduck** [comment]
Automated screenshot from https://beta.duckduckgo.com
                \nmobile: https://ia-screenshots.s3.amazonaws.com/scratch_cheat_sheet_mobile.png

**Comment by moollaza** [comment]
@preemeijer this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by moollaza** [comment]
@preemeijer congrats! It's live: https://duckduckgo.com/?q=scratch+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**